www/templates/customer-balance.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  <p>Balance: {{customer.balance}} rCredits</p>
+  <p>Rewards: {{customer.rewards}} rCredits</p>
```

Comment:
We would sometimes say "$12.34r", but in this case I think we would just use "$12.34". The "r" is only necessary when distinguishing from USD.